### PR TITLE
silence a warning in msvc 19

### PR DESCRIPTION
**Pull request by walkingeyerobot**
LOOK I'M HELPING.

### CODE REVIEW

File: nsasm/parse.cc [hunk 1]
```diff
@@ -398,6 +398,7 @@ ErrorOr<Directive> ParseDirective(TokenSpan* pos) {
       return std::move(directive);
     }
   }
+  return Error("unknown directive_type: %d", directive_type);
```
Comment (7sempra) [inline nsasm/parse.cc:401]: This is sad; it means that the compiler will no longer protect against new entries being added to the enum.

Comment (vslashg) [reply nsasm/parse.cc:401]: Yeah, this is a pessimisation for the compiler that we're actively developing this on.

**Comment by vslashg** [comment]
I'm happy to consider MSVC correctness patches, and if a developer comes along who wants to contribute changes and use MSVC 2017 or later to do it, I'd certainly reconsider this issue.

For now, we care about this working on Windows, but not so much the dev experience there.

**Comment by vslashg** [comment]
In retrospect this doesn't harm anything in clang (like adding a default: case would have), so we may as well accept it.

**Pull request closed without merging**